Lab1ForReview/Lab1b/Lexic.txt

```diff
@@ -0,0 +1,31 @@
+
+
+Alphabet:
+
+a.Upper (A-Z) and lower case letters (a-z) of the English alphabet
+b. Decimal digits (0-9);
+
+1. Lexic:
+ a.Special symbols, representing: 
+	-operators + - * / > < == =< => =!
+	-separators () {} | space ' "
+	-reserved keywords: nr NR boolean string ret floop wloop daca altfel false true in out
+ b.identifiers:
+	A sequence of letter and digits with the first character being a letter
+	identifier ::= letter | letter{letter}{digit}
+ 	letter ::= "A" | "B" | . ..| "Z"
+ 	digit ::= "0" | "1" |...| "9" 
+ c.constants 
+	1.interger 
+	noconst := |"-"no|no !
```

Comment:
For example the definition connector used for constants ( := ) is neither in BNF nor in EBNF.